src/ng/parse.js

```diff
@@ -1950,23 +1956,20 @@ function $ParseProvider() {
 
       var fn = parsedExpression.oneTime ? oneTimeInterceptedExpression : regularInterceptedExpression;
 
-      // Propogate the literal/oneTime attributes
+      // Propogate the literal/oneTime/constant attributes
       fn.literal = parsedExpression.literal;
       fn.oneTime = parsedExpression.oneTime;
+      fn.constant = parsedExpression.constant;
 
-      // Propagate or create inputs / $$watchDelegates
-      useInputs = !parsedExpression.inputs;
-      if (watchDelegate && watchDelegate !== inputsWatchDelegate) {
```

Comment:
I see. And we are fine with that because we never use `$$watchDelegate` with interceptors in core, right?
I might add a comment (where?) about that, i.e. the you can't use `$$watchDelegate` with interceptors.